### PR TITLE
Changed font to OpenSans

### PR DESCRIPTION
**Pull request by ahmad2shadab**
**Font changed** to *openSans*

### CODE REVIEW

File: app/globals.css [hunk 1]
```diff
@@ -2,12 +2,6 @@
 @tailwind components;
 @tailwind utilities;
  
-html,
```
Comment (hadissulejmani) [inline app/globals.css:5]: Please do not remove lines of code that are written by other developers, if it is not included in the task requirements. This part of code is important and should stay as it is!

Comment (hadissulejmani) [reply app/globals.css:5]: Please make a new commit with the previous code from app/globals.css!

Comment (ahmad2shadab) [reply app/globals.css:5]: I don't know how the previous code got deleted, but anyway I've created a PR with the the previous code for **app/global.css** from `bugfix/font_family_change`

Comment (hadissulejmani) [reply app/globals.css:5]: Okay I will check it!

**Pull request merged**